cmake/Config.cmake.in

```diff
@@ -2,3 +2,8 @@
 
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
 check_required_components("@PROJECT_NAME@")
+
+include(CMakeFindDependencyMacro)
+if (@WITH_OPENSSL@)
+    find_dependency(OpenSSL)
+endif()
```

Comment:
This needs to be done for all the different types of TLS libraries.  Suggest
````
diff --git a/cmake/Config.cmake.in b/cmake/Config.cmake.in
index 9c15f36a..a7faa72f 100644
--- a/cmake/Config.cmake.in
+++ b/cmake/Config.cmake.in
@@ -2,3 +2,20 @@

 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
 check_required_components("@PROJECT_NAME@")
+
+include(CMakeFindDependencyMacro)
+if (@WITH_GNUTLS@)
+    find_dependency(GnuTLS)
+endif()
+if (@WITH_OPENSSL@)
+    find_dependency(OpenSSL)
+endif()
+if (@WITH_MBEDTLS@)
+    find_dependency(MbedTLS)
+endif()
+if (@WITH_WOLFSSL@)
+    find_dependency(WolfSSL)
+endif()
+if (@WITH_TINYDTLS@ AND NOT @USE_VENDORED_TINYDTLS@)
+    find_dependency(TinyDTLS)
+endif()
````

Reply:
makes sense, done